docs/download.html

```diff
@@ -0,0 +1,546 @@
+        
+          If you prefer to run oVirt Engine standalone on physical hardware, you can install oVirt Engine and Nodes / Hosts separately. See the oVirt Installation Guide for full installation instructions for this configuration.
+        </p> 
+    </section>
+
+    <section class="ov-layout-styles ov-content-styles">
+      <h2 class="ov-heading ov-heading--lg">Download and Install Self-Hosted Engine</h2>
+      <div>
+        <p>
+        For production installations, we recommend installing oVirt using the Self-Hosted Engine configuration. In this configuration, oVirt Engine and a Host are installed together with the Engine running as a Virtual Machine on that Host. This configuration is more resilent because the Engine Virtual Machine will be highly available (once a second Host is added). See the Self-Hosted Engine Guide for full installation instructions for this configuration.
+
+        oVirt Self-Hosted Engine is installed using a graphical installer in Cockpit. Cockpit is enabled by default on oVirt Nodes (see below). If you are using a Enterprise Linux Host, see Installing Cockpit on Enterprise Linux Hosts in the Installation Guide.
+        </p>  
+      </div>  
+    </section>
+
+    <section class="ov-layout-styles ov-content-styles">
+      <h2 class="ov-heading ov-heading--lg">Red Hat Enterprise Linux, CentOS Linux</h2>
+      <div>
+       <ol>
+         <li> Enable the Base, Optional, and Extra repositories (they are enabled by default on CentOS). </li>
+         On Red Hat Enterprise Linux you'll need a valid subscription and the following repositories enabled:
+         <ul>
+            <li>rhel-7-server-rpms</li>
+            <li>rhel-7-server-optional-rpms</li>
+            <li>rhel-7-server-extras-rpms</li>
+          </ul>
+         <li>Add the official oVirt repository.</li> 
+         (add code snippet)
+         <li>Enable Cockpit.</li> 
+         (add code snippet)
+         <li>Install cockpit-ovirt-dashboard.</li> 
+         (add code snippet)
+       </ol>
+      </div>  
+    </section>
+
+    <section class="ov-layout-styles ov-content-styles">
+      <h2 class="ov-heading ov-heading--lg">Install oVirt Self-Hosted Engine</h2>
+      <div>
+       <ol>
+         <li>Log in to Cockpit at https://[Host IP or FQDN]:9090 and click Virtualization → Hosted Engine.</li>
+         <li>Click Start under the Hosted Engine option.</li>
+         <li>Complete the setup wizard. As part of the setup, you will enter the Hosted Engine’s name.</li>
+         <li>Once the installation completes, oVirt’s web UI management interface will start. Browse to https://[Hosted Engine’s name]/ to begin using oVirt!</li>
+            *See Browsers and Mobile Clients for supported browsers and mobile client information.*
+        </ol>
+      </div>
+    </section>
+
+    <section class="ov-layout-styles ov-content-styles">
```

Comment:
I wonder if it makes more sense to have one wrapper around this whole content area and put the `ov-layout-styles`, `ov-content-styles` on that and then just styles the section tags to have the right spacers. WDYT?

Reply:
Are you thinking of just putting everything in one big section/ content area and then applying the `ov-layout-styles` and `ov-content-styles` to that big section? That would probably help to simplify things.

Reply:
ya I think that will be much simpler and better for other pages too.

Reply:
Cool yeah I think that's a good idea. I can update the community page too so it follows a similar class format.